html/controls/operator.js

```diff
@@ -166,6 +166,15 @@ function createMetaControlTable() {
 			$("tr.SpeedScore").toggleClass("Show", this.checked);
 		});
 
+	$("<label>").text("Show Start/Stop Buttons").attr("for", "ShowClockControlsButton")
+		.appendTo(buttonsTd);
+	$("<input type='checkbox'>").attr("id", "ShowClockControlsButton")
+		.appendTo(buttonsTd)
+		.button()
+		.click(function() {
+			$("tr.Control").toggleClass("Show", this.checked);
```

Comment:
Can this not affect tr.Controls on other tabs?

Reply:
Quick check shows that it does.
Since I copied this code from the other show/hide buttons, the problem extends to all of them. (Though the other ones affect elements with less generic class names.)

Reply:
Restricted the search to the #TeamTime tab for all three buttons.

(At my current rate of learning, I can add jquery and css to the list of skills in my CV before the end of the year.)